src/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from "classnames";
+import { Link, useLocation } from "react-router-dom";
+import { Address } from "../Pages/Home/Home";
+import { abbreviateBoro, toTitleCase } from "../../helpers";
+import "./ProgressBar.scss";
+
+type Step = { path?: string; name?: string; active?: boolean };
+type ProgressBarProps = {
+  address?: Address;
+  lastStepReached?: number;
+};
+
+export const ProgressBar: React.FC<ProgressBarProps> = ({
+  address,
+  lastStepReached = -1,
+}) => {
+  const location = useLocation();
+  const steps = makeProgressSteps(location.pathname, address);
+
+  const progressBarSteps = steps.flatMap((step, index, array) => {
+    const StepText =
+      step.active || index > lastStepReached ? (
+        <span>{step.name}</span>
+      ) : (
+        <Link to={step.path ?? "/"} className="jfcl-link">
+          {step.name}
+        </Link>
+      );
+
+    const Step = (
+      <div
+        key={index}
+        className={classNames("progress-bar__step", step.active && "active")}
+      >
+        <div className="progress-bar__step__number">{index + 1}</div>
+        {StepText}
+      </div>
+    );
+
+    if (index !== array.length - 1) {
+      return [Step, <hr key={"hr-" + index} />];
```

Comment:
really like the way you handled the dividing lines with `<hr>`!

Reply:
thanks! I always forget about some of those extra tags but can be so handy